### ISSUE TITLE
NEON site data requirements 

### ISSUE THREAD

**Issue by jedwards4b** [ESCOMP/CTSM#1443]
There are still several global data sets being downloaded for the single point NEON data sites.

- fndep_clm_f09_g17.CMIP6-SSP3-7.0-WACCM_1849-2101_monthly_c191007.nc
- aerodep_clm_SSP370_b.e21.BWSSP370cmip6.f09_g17.CMIP6-SSP3-7.0-WACCM.001_2014-2101_monthly_0.9x1.25_c190402.nc
- clmi.I2000Clm50BgcCrop.2011-01-01.1.9x2.5_gx1v7_gl4_simyr2000_c190312.nc

Can we subset these as we did the lightning data?

**Comment by ekluzek** [comment]
Good question @jedwards4b. The first two you could with no problems as they are on a regular grid. The first is managed by CTSM and the second by cdeps, but if you add these to the NEON user-mod directory settings it should handle it appropriately.

The last one won't work as it's on a vector format and it's the model initial conditions. Eventually you are going to have initial conditions for NEON sites, so there is no reason to this right now. When you have initial condition files for NEON sites that will be taken care of at that point. An interim solution we could go to though would be to use a lower resolution dataset rather than 1-degree. But, that also means that it won't be spun up as well. So I think you just wait until you have IC files for NEON sites.

**Comment by jedwards4b** [comment]
@ekluzek thanks for the guidance.  I'll create subsets of the first two files.

**Comment by wwieder** [comment]
Has this been updated in the latest CESM-Lab image @negin513 and @danicalombardozzi? Running CESM-Lab with `docker pull escomp/ctsm-lab-2.3-preview` My code is still getting hung up at this step, which is taking a _very_ long time.

`Model ctsm missing file stream_fldfilename_ndep = '/home/user/inputdata/lnd/clm2/ndepdata/fndep_clm_f09_g17.CMIP6-SSP3-7.0-WACCM_1849-2101_monthly_c191007.nc'
Trying to download file: 'lnd/clm2/ndepdata/fndep_clm_f09_g17.CMIP6-SSP3-7.0-WACCM_1849-2101_monthly_c191007.nc' to path '/home/user/inputdata/lnd/clm2/ndepdata/fndep_clm_f09_g17.CMIP6-SSP3-7.0-WACCM_1849-2101_monthly_c191007.nc' using WGET protocol.`

**Comment by danicalombardozzi** [comment]
> The other file I'm considering is fndep_clm_f09_g17.CMIP6-SSP3-7.0-WACCM_1849-2101_monthly_c191007.nc it's 4GB , I've reduced to 222MB in fndep_clm_f09_g17.CMIP6-SSP3-7.0-WACCM_2018-2030_monthly_c210826.nc

Hi @jedwards4b: It looks like the container is still trying to download the fndep_clm_f09_g17.CMIP6-SSP3-7.0-WACCM_1849-2101_monthly_c191007.nc file. I know you created a smaller version, but it seems that the code isn't pointing to that?

**Comment by ekluzek** [comment]
@danicalombardozzi I verified that's true of the "neon ui" branch. But, it should be easy to add to the branch. It just needs to go into the NEON user-mod directory. I should be able to figure that out and add it in. Jim is out this week, but he could also add it in when he gets back. If needed we could make a branch tag again so the container could be updated with it as well.

**Comment by ekluzek** [comment]
I thought Jim might've had a file that was created, but just wasn't pointed to. But, I'm not seeing it anywhere, so we'll wait until Jim is back to do this. He was just getting a subset of the file, spatially and temporally, so he wasn't doing anything fancy or hard to do, but I'll let him do it.

**Comment by danicalombardozzi** [comment]
Thanks for the clarification @ekluzek! It seems that @jedwards4b needs to create the file (or at least let us know where it is on Cheyenne) so that we can point to it. We are planning to send this initial version out to testers tomorrow, so this will need to come in with the next round of changes. It's taking me >1 hour to download the file (slow mountain internet), but even folks with faster internet are still having to wait ~30 minutes.

**Comment by briandobbins** [comment]
Just to add to this, I'll look at adding the files needed for these cases
to the secondary inputdata server, in case the first is just getting
hammered and that's contributing to the speed issues.  I'll do a speed test
tonight on that and see what I find.  It won't help if the issue is due to
one's internet speed, but will if it's due to server bandwidth.  Once the
new, smaller file is made, we can put that on both servers, too.

- Brian

**Comment by wwieder** [comment]
@jedwards4b are you able to update to the smaller ndep file you created for NEON simulations?

**Comment by jedwards4b** [comment]
@wwieder yes I've now updated the ctsm branch with that change, sorry I missed it before.

**Comment by wwieder** [comment]
@briandobbins is this something that's easy enough to pull into CESM-Lab now, or are we better off to stack up a few changes (e.g. surface datasets & updated initial conditions files that @negin513 will do) for a larger update in a bit more time?

**Comment by wwieder** [comment]
in #1467
